test/components/core_spec.js

```diff
@@ -95,22 +97,25 @@ describe('Core', function() {
           expect(fullScreenSpy).to.have.been.calledWith(this.core.activeContainer.el)
         })
       })
+
```

Comment:
There are a number of those empty lines at the end of blocks, please remove those.